packages/orchestration/src/orchestration-client.ts

```diff
@@ -34,14 +38,18 @@ const logger = createLogger({
  * Get the orchestration client.
  */
 export class OrchestrationClient {
+  private useClientHistory = true;
+  private messagesHistory?: ChatMessages;
   /**
    * Creates an instance of the orchestration client.
    * @param config - Orchestration module configuration. This can either be an `OrchestrationModuleConfig` object or a JSON string obtained from AI Launchpad.
+   * @param clientConfig - Client configuration for the orchestration client.
    * @param deploymentConfig - Deployment configuration.
    * @param destination - The destination to use for the request.
    */
   constructor(
     private config: OrchestrationModuleConfig | string,
+    clientConfig?: ClientConfig,
```

Comment:
[pp] Adding one more params in the constructor is breaking and looks not really elegant. Since this feature is probably only used in our LangChain Orchestration client as an opt-in, maybe we can add a setter for this and then in LangChain Orchestration, we add a clientConfig option, and call the setter internally in LangChain Orchestration after the client initialization?

Reply:
This is actually not used in langchain at all.
I agree that adding another param is annoying, but I do not see a more elegant way, other than creating some consolidating object as an in-between layer.